### PR TITLE
[fe] Apps Icon Package

### PR DESCRIPTION
**Pull request by Jordan-Mysten**
This introduces a new package, `@mysten/icons`, which exports all of the SVG icons exported from the Icons Figma file. These icons can directly be imported from projects using named imports. Unused icons should be tree-shaken out.

The svgs are pre-built in React components, so they can easily be used in any project. We should be able to easily share the components between explorer and wallet without any additional work.

This approach includes a lot of generated code being checked in (~4k lines), and I'd be happy to talk through if this is the best way to do this. We could also avoid checking it in and make it solely a build artifact, although I generally prefer to have these kinds of things checked-in.

### CODE REVIEW

File: apps/icons/svgrrc.config.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+// SPDX-License-Identifier: Apache-2.0
+`;
+
+/** @type {import('@svgr/core').Config} */
+module.exports = {
+  icon: true,
+  typescript: true,
+  outDir: "./src",
+  jsxRuntime: "automatic",
+  replaceAttrValues: {
```
Comment (mamos-mysten) [inline apps/icons/svgrrc.config.js:17]: 👌🏼

Comment (mamos-mysten) [reply apps/icons/svgrrc.config.js:17]:
```
  svgoConfig: {
    plugins: [{ name: "convertColors", params: { currentColor: true } }],
  },
```
will achieve the same thing, i believe, without having to replace a specific value

**Review by mamos-mysten — APPROVED** [review]
This approach has worked well for me in the past, and implementation looks 👌🏼 to me.

File: apps/icons/svgrrc.config.js (hunk 1 above)
Comment (Jordan-Mysten) [reply apps/icons/svgrrc.config.js:17]: Sadly I think that'll fail for colors used in clip paths.

**Pull request merged**